sys/net/network_layer/sixlowpan/ip.c

```diff
@@ -33,6 +33,7 @@
 #include "lowpan.h"
```

Comment:
Does it also make sense to outsource the changes to this diff to its own PR? Doesn't seem like that there are any dependencies to your RPL refactorings?